src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount = 0;
+
+  if (days < 3) {
+    amount = 40 * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    amount = 40 * days - 20;
+  }
+
+  if (days >= 7) {
+    amount = 40 * days - 50;
+  }
```

Comment:
40, 50, and 20 are magic numbers
try to keep all numbers like these in variables for more readable code